swagger.yaml

```diff
@@ -3786,6 +3786,64 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /v2/clusters/{cluster_id}/ui-settings:
+    get:
+      tags:
+        - installer
+      description: Fetch cluster specific installer UI settings.
+      operationId: v2GetClusterInstallerUISettings
+      parameters:
+        - in: path
+          name: cluster_id
+          description: The cluster for which UI settings should be retrieved.
+          type: string
+          format: uuid
+          required: true
+      responses:
```

Comment:
Shouldn't we also be including 401 at least, maybe also 403?

Reply:
Done